src/configurations/sources/shopify/ui-config.json

```diff
@@ -3,11 +3,36 @@
     {
       "title": "Connection Settings",
       "fields": [
+        {
+          "type": "singleSelect",
+          "label": "Version of app",
+          "value": "version",
+          "options": [
+            {
+              "name": "Client side tracking with the Rudderstack tracker script (Legacy).",
+              "value": "v1"
+            },
+            {
+              "name": "Client side tracking with the Rudderstack web pixel (Recommended).",
+              "value": "v2"
+            }
+          ],
+          "defaultOption": {
+            "name": "tracker",
+            "value": "v1"
```

Comment:
The values rather than `v1/2` can we keep `legacy / `pixel` ?

Reply:
Can you once confirm with KC/Varun

Reply:
yes talked with KC on this